tests/unit/test_props_parser.py

```diff
@@ -0,0 +1,478 @@
+from collections import namedtuple
+import pytest
+from unittest.mock import MagicMock, call
+
+# helpers variables to make test input/outup easier to change
+FIELD = "field"
+FIELDS = f"{FIELD}s"
+FIELD1 = f"{FIELD}1"
+FIELD2 = f"{FIELD}2"
+FIELD3 = f"{FIELD}3"
+FIELD4 = f"{FIELD}4"
+FIELD5 = f"{FIELD}5"
+FIELD6 = f"{FIELD}6"
+FIELD7 = f"{FIELD}7"
+FIELD8 = f"{FIELD}8"
+NAME = "Name"
+OUTPUTNEW = "OUTPUTNEW"
+OUTPUT = "OUTPUT"
+INPUT_FIELDS = "input_fields"
+OUTPUT_FIELDS = "output_fields"
+STANZA = "stanza"
+LOOKUP_STANZA = f"lookup_{STANZA}"
+STANZA_TYPE = f"{STANZA}_type"
+CLASSNAME = "classname"
+
+PropsProperty = namedtuple("PropsProperty", ["name", "value"])
+
+
+@pytest.fixture
+def field_mock(monkeypatch):
+    field = MagicMock()
+    monkeypatch.setattr(
+        "pytest_splunk_addon.standard_lib.addon_parser.props_parser.Field", field
+    )
+    return field
```

Comment:
instead of above you can mock `convert_to_fields` (or anything you need) and reload your module. It will work when pytest_splunk_addon is loaded before test, if we need to run tests without already loaded psa, then need some `if` or `try` to handle this 
```suggestion
with patch.object(sys.modules['pytest_splunk_addon.standard_lib.addon_parser'], 'convert_to_fields', lambda x: x):
    import pytest_splunk_addon.standard_lib.addon_parser.props_parser
    importlib.reload(pytest_splunk_addon.standard_lib.addon_parser.props_parser)
```

Reply:
Done

Reply:
Nice catch!